docker/encfs/Dockerfile.encfs

```diff
@@ -7,9 +7,9 @@ RUN cd tools/get-snp-report && make && mv bin/get-snp-report /
 RUN CGO_ENABLED=0 GOOS=linux go build -o / ./cmd/azmount
 RUN CGO_ENABLED=0 GOOS=linux go build -o / ./cmd/remotefs
 
-FROM alpine:3.18.6
+FROM mcr.microsoft.com/cbl-mariner/base/core:2.0
 
-RUN apk update && apk upgrade --no-cache && apk add --no-cache cryptsetup fuse3 curl bash jq
+RUN tdnf update -y && tdnf upgrade -y && tdnf install -y cryptsetup fuse jq

```

Comment:
add `&& tdnf clean all` at the end to remove cache files we don't need stored in the layer.